File-Formats/PDF/Convert-HTML-To-PDF/mac.md

```diff
@@ -9,75 +9,73 @@ keywords: Assemblies
 
 # Convert HTML to PDF file in Mac
 
-Steps to convert HTML to PDF in ASP.NET Core Mac:
+## Steps to convert HTML to PDF in ASP.NET Core Mac:
```

Comment:
Pls make sure whether you have created the project in ASP.NET Core or ASP.NET MVC application and modify it

Reply:
Changed in Asp.net core mvc